rcl/include/rcl/arguments.h

```diff
@@ -34,6 +34,9 @@ typedef struct rcl_arguments_t
   struct rcl_arguments_impl_t * impl;
 } rcl_arguments_t;
 
+#define RCL_LOG_LEVEL_ARG_RULE "__log:="
```

Comment:
In ROS 1 `__log:=` is a [path to a log file that the node to writes to code ](https://github.com/ros/ros_comm/blob/e96c407c64e1c17b0dd2bb85b67f388380527097/clients/roscpp/src/libros/file_log.cpp#L63-L67). Maybe using a different name for log level would help someone porting a tool that uses `__log` notice the difference?

Reply:
yep, sounds like a good idea. I was going off usage of other command line tools but missed considering arguably the most important one :smile: